compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/window/Dialog.skiko.kt

```diff
@@ -192,14 +192,15 @@ private fun DialogLayout(
     content: @Composable () -> Unit
 ) {
     val currentContent by rememberUpdatedState(content)
-    val platformInsets = properties.platformInsets
+
     val layer = rememberComposeSceneLayer(
         focusable = true
     )
     layer.scrimColor = properties.scrimColor
     layer.setKeyEventListener(onPreviewKeyEvent, onKeyEvent)
     layer.setOutsidePointerEventListener(onOutsidePointerEvent)
     layer.Content {
+        val platformInsets = properties.platformInsets
```

Comment:
Why do you need this change? Let's try to reduce the scope and don't touch the code outside iOS

Reply:
Because platformInset end up accessing CompositionLocal SafeArea of parent composition, which is unrelated.

Reply:
Probably makes sense 🤔 
Could you please add an explicit comment (or better some test)?

Reply:
Example: dialog from a small compose view inside the app that doesn't intersect any safe areas, won't have them either. 

`layer.Content` uses composition locals provided by mediator that knows about proper safe-areas of view where dialog resides